### ISSUE TITLE
Downloading the doc resources currently broken with HTTP 401

### ISSUE THREAD

**Issue by mle-idealo** [spring-io/spring-doc-resources#71]
Our CI builds are currently failing while trying to download the doc resources:

`
Error:  Failed to execute goal com.googlecode.maven-download-plugin:download-maven-plugin:1.7.0:wget (unpack-doc-resources) on project offer-attribute-assignor: Download failed with code 401
`

The error showed up starting May 25th.

Was there any change on your side? Could you assist?

**Comment by bclozel** [comment]
This is due to [policy changes on our release repository](https://status.spring.io/incidents/rl38k61jk6hr) as [announced a few months ago](https://spring.io/blog/2022/12/14/notice-of-permissions-changes-to-repo-spring-io-january-2023).

As you've probably seen, we've stopped investing in this repository in favor of [spring-asciidoctor-banckends](https://github.com/spring-io/spring-asciidoctor-backends) and Antora (Spring Framework and Spring Security ref docs now use it). I don't think we are going to release a new version of this project to Maven Central as it would be sending the wrong signal: we don't intend to support this project in the long run and it was mostly designed for internal usage in the Spring team.

There are two ways to fix your build:
* upgrade to spring-asciidoctor-banckends, see [an example of that](https://github.com/spring-projects/spring-graphql/commit/9073d0d7cebd8880251050f2c82582c7de2f559a)
* we could release a new version on the milestone repository; but this would also suggest that we will ship a new release as well

Let us know what you think!

**Comment by mle-idealo** [comment]
Heyho Brian, thanks for the quick response. Actually I've had Antora on my personal road map since some time but did not yet manage to take a closer look into it.
Obviously the time has come now. Since the adoption in our (Maven) project is fairly small, we'll absolutely take this challenge and migrate forward together with your Framework and Security docs.

Regarding the main goal of internal usage: Well, what good is for the Spring teams is always also good for us. ;-)

Long story short: We'll move forward to the new asciidoctor backends as mentioned in your link above.

**Comment by bclozel** [comment]
The "spring-asciidoctor-backends" project is not using antora at all. It provides an enhanced experience compared to this project but does not solve the publication of docs for multiple versions. I think using it is a nice first step to get your project back on track.

If you're looking for our opinionated view of Antora, you can [start with this project](https://github.com/spring-gradle-plugins/spring-antora-plugin). I'm not familiar enough with the Antora project yet, but I think that considering plain Antora and adding your own theme and preferences is a perfectly viable choice here.

Thanks!
